tests/microbenchmarks/skip_softmax/bench_skip_softmax_fmha_exe.py

```
@@ -0,0 +1,275 @@
+                   warm_up: int) -> List[dict]:
+    rows: List[dict] = []
+    for cfg in cfgs:
+        for thr in cfg.threshold_sweep:
+            cmd = _build_cmd(fmha_exe,
+                             cfg,
+                             thr,
+                             with_stat=True,
+                             runs=1,
+                             warm_up=warm_up)
+            print(f"[pass1] {cfg.name} threshold={thr:g} ...", flush=True)
+            elapsed, skip = _run_once(cmd)
```

Ruff (0.15.12)

[warning] Unpacked variable `elapsed` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)